### PR TITLE
adds getAllChunksInRegion function

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

File: src/anvil.js [hunk 1]
```diff
@@ -48,6 +48,20 @@ module.exports = (mcVersion) => {
...
       await region.write(x & 0x1F, z & 0x1F, nbt)
     }
+
+    async getAllChunksInRegion (x, z) {
+      const region = await this.getRegion(x * 32, z * 32)
+      const chunks = []
+      for (let _x = 0; _x < 32; _x++) {
+        for (let _z = 0; _z < 32; _z++) {
+          if (region.hasChunk(_x, _z)) {
+            chunks.push(await this.load(x * 32 + _x, z * 32 + _z))
```
Comment (rom1504) [inline src/anvil.js]: does it work in parallel ? promise map

Comment (u9g) [reply src/anvil.js]: done

**Review by rom1504 — CHANGES_REQUESTED** [review]
can you update the doc with the new function

**Comment by u9g** [comment]
> can you update the doc with the new function

done!

**Review by rom1504 — APPROVED** [review]

**Comment by rom1504** [comment]
might want to add a getAllRegions functions and an example on how to get all chunks of all regions ?

**Comment by u9g** [comment]
> might want to add a getAllRegions functions and an example on how to get all chunks of all regions ?

regions are just the x z in the fileNames of the files made by the world generator, its just a matter of matching the file names with regex, to get the xz for the all regions, but I can add that too

**Pull request merged**